src/streamlink/plugins/mediavitrina.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import validate
+from streamlink.stream import HLSStream
+from streamlink.utils import parse_json
+from streamlink.utils.url import update_qsd
+
+log = logging.getLogger(__name__)
+
+
+class MediaVitrina(Plugin):
+    _re_url_1 = re.compile(r'https?://(?P<channel>ctc(?:love)?|chetv|domashniy|5-tv)\.ru/(?:online|live)')
+    _re_url_2 = re.compile(r'https?://(?P<channel>ren)\.tv/live')
+    _re_url_3 = re.compile(r'https?://player\.mediavitrina\.ru/(?P<channel>[^/?]+.)(?:/[^/]+)?/[\w_]+/player\.html')
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return (
+            cls._re_url_1.match(url) is not None
+            or cls._re_url_2.match(url) is not None
+            or cls._re_url_3.match(url) is not None
+        )
```

Comment:
Not sure if this should be changed to a single regex.

If we ever switch the plugin matching system and load plugins on demand with a list of regexes extracted from the plugin modules during build time, then we need to implement a system that can take multiple regexes into account for a single plugin. There are only a handful of plugins currently (5 or so) that use "extended" logic in the `can_handle_url` method.

Shouldn't be too much of an issue though, so feel free to ignore.